packages/ui/src/components/editor/panels/Hierarchy/node/index.tsx

```diff
@@ -312,8 +312,14 @@ export const HierarchyTreeNode = (props: HierarchyTreeNodeProps) => {
     <li
       style={props.style}
       className={twMerge(
-        `bg-${props.index % 2 ? 'theme-surfaceInput' : 'zinc-800'}`,
-        selected ? 'border border-gray-100' : 'border-none'
+        `${
+          props.index % 2
+            ? `${selected ? 'bg-[#1d1f23]' : 'bg-theme-surfaceInput'} hover:bg-[#1d1f23]`
+            : `${selected ? 'bg-zinc-700' : 'bg-zinc-800'} hover:bg-zinc-700`
+        }`,
```

Comment:
```suggestion
          props.index % 2
            ? (selected ? 'bg-[#1d1f23]' : 'bg-theme-surfaceInput'} hover:bg-[#1d1f23]')
            : (selected ? 'bg-zinc-700' : 'bg-zinc-800'} hover:bg-zinc-700'),
```

twMerge works on `any` type. also just the final result needs to be string which in this case already is

Reply:
oh, cool works either way, the latter is more easier to read, lets go with that